### ISSUE TITLE
kops get cluster fails after cluster state is changed where kops update is done

### ISSUE THREAD

**Issue by jungwookim** [kubernetes/kops#9745]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**
`Version 1.120 (git-2c2042465)`
**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**
```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.6-beta.0", GitCommit:"e7f962ba86f4ce7033828210ca3556393c377bcc", GitTreeState:"clean", BuildDate:"2020-01-15T08:26:26Z", GoVersion:"go1.13.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.9", GitCommit:"16236ce91790d4c75b79f6ce96841db1c843e7d2", GitTreeState:"clean", BuildDate:"2019-03-25T06:30:48Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```
**3. What cloud provider are you using?**
aws
**4. What commands did you run?  What is the simplest way to reproduce this issue?**
(with kops v1.18.6)
```
kops get clusters
kops upgrade cluster $my_cluster --yes
kops update cluster $my_cluster --yes
kops rolling-update cluster $my_cluster
```
I means I was trying to update OLD: 1.11.9 to NEW 1.18.6 (It was so stupid)

So, at that moment, I was trying to command `k8s get $my_cluster` but it didn't work on version 1.12.0
I found out https://github.com/kubernetes/kops/issues/7806#issuecomment-543272778 and I replaced `apiVersion` in config and files in instancegroup.

**5. What happened after the commands executed?**
1. no kind "Cluster" is registered for version "kops.k8s.io/v1alpha2" in scheme "k8s.io/kops/pkg/kopscodecs/codecs.go:33"
(after changing config and files in instancegroup) 2. AccessDenied: Access Denied
**6. What did you expect to happen?**
I'd like to get `kops get $my_cluster`
**7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.**

```yaml
```

**8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.**

**9. Anything else do we need to know?**
As I said, I was too stupid to upgrade kops versions.
So, basically my kops was version 1.11.9 but my local kops version was 1.18.6. Then I have tried to upgrade. At that time, it was messed up. So I found the similar issue #7806 which says i need to edit `apiVersion` from `kops.k8s.io/v1alpha2` to `kops/v1alpha2`. After then, still I can't get clusters with error `access denied` even if I have authorities.

I'd like to revert state into 1.11.9 and then try to upgrade bit by bit

**Comment by hakman** [comment]
Kops 1.18 should still work ok with Kubernetes 1.12. Could you just try to set the `kubernetesVersion` and update & rolling-update your cluster?
```yaml
spec:
  kubernetesVersion: 1.11.9
```

**Comment by jungwookim** [comment]
@hakman I've edited `spec/kubernetesVersion` and `kops_1.12.0 get clusters` showed me $my_cluster but `kops_1.12.0 get $my_cluster` access denied with a reason `error reading InstanceGroup "master-us-east-1a"` that's weired

**Comment by hakman** [comment]
I cannot help you with editing. I just suggested to use Kops 1.18 to downgrade Kubernetes to 1.11 or what version you had before.

**Comment by jungwookim** [comment]
@hakman Thank you and tried to downdgrade by using Kops 1.18 and still can't fetch master node. That's the problem. I'll figure it out more.

**Comment by hakman** [comment]
If you did and worked, you can just keep using Kops 1.18 to manage it. Kops 1.18 can work with any Kubernetes 1.10+.

**Comment by jungwookim** [comment]
It still didn't work. I got access denied `instancegroup/master-us-east-1a: AccessDenied: Access Denied`from S3 when I command `kops_1.12.0 get $my_cluster` whiist `kops_1.12.0 get clusters` shows the cluster. I'm investigating why.
